cli/src/main/java/com/box/l10n/mojito/cli/command/extractioncheck/ExtractionCheckNotificationSenderGithub.java

```diff
@@ -118,6 +118,15 @@ public void sendChecksSkippedNotification() {
       githubClients
           .getClient(githubOwner)
           .addCommentToPR(githubRepo, prNumber, GithubIcon.WARNING + " " + checksSkippedMessage);
+      githubClients
```

Comment:
is it relevant for this action to be in the `if` statement? This should be done regardless of the message, no?

Reply:
Thank you for pointing that out. Looking at that closely, I moved the statement to a higher level method.